src/controller/CHIPDeviceController.cpp

```diff
@@ -1702,10 +1789,12 @@ void DeviceCommissioner::AdvanceCommissioningStage(CHIP_ERROR err)
         ChipLogProgress(Controller, "Arming failsafe");
         // TODO(cecille): Find a way to enumerate the clusters here.
         GeneralCommissioningCluster genCom;
-        uint16_t commissioningExpirySeconds = 5;
         // TODO: should get the endpoint information from the descriptor cluster.
         genCom.Associate(device, 0);
+#if !CHIP_DEVICE_CONFIG_ENABLE_BOTH_COMMISSIONER_AND_COMMISSIONEE // temporary - until example app clusters are updated (Issue 8347)
+        uint16_t commissioningExpirySeconds = 5;
         genCom.ArmFailSafe(mSuccess.Cancel(), mFailure.Cancel(), commissioningExpirySeconds, breadcrumb, kCommandTimeoutMs);
```

Comment:
If these are turned off, how does the commissioner get into the next state? The state change is triggered from the callbacks.

Reply:
Commissioning does not work yet when this is turned on - that is my next step to make work (as part of issue 8347). Just trying to get this first part in which includes UDC client/server which uses the resolver feature of DeviceCommissioner.

Reply:
Sure, sounds fine.